dev/script/generate-dot-properties/service-config.js

```diff
@@ -120,6 +120,7 @@ function getMediaApiConfig(config) {
         |quota.store.key=rcs-quota.json
         |security.cors.allowedOrigins=${getCorsAllowedOriginString(config)}
         |metrics.request.enabled=false
+        |image.record.download=false
```

Comment:
Maybe this should be true for dev?

Reply:
Are you thinking it might be useful for something in particular when developing locally?